examples/qec_decoder_toolkit/Quantinuum_hseries_qec_decoder_toolkit.ipynb

```diff
@@ -250,9 +250,9 @@
     "        * Random number generation\n",
     "        * File I/O\n",
     "        * Exception Handling\n",
-    "1. Quantum programs can only call Wasm functions that accept multiple integers and return an integer. It is not possible to use void Wasm functions, since this is not supported by the hardware.\n",
-    "    * For example, examine the call `x = wasm_func(a, b, c);` The `wasm_func` function can exist in QASM where `a`, `b`, `c`, and `x` are classical registers. \n",
-    "    * Such function calls have timing restrictions. Wasm calls in quantum programs have been run that took multiple milliseconds.\n",
+    "1. Quantum programs can call two types of Wasm functions: \n",
+    "    * Void Wasm functions, e.g. to initialize the Wasm environment.\n",
```

Comment:
I would like to get clarification on this point from the hardware team, to make sure this is all fine. The question that is left is: are void functions allowed in the middle of the circuit and is it allowed to call the init function in the middle of the circuit.

Reply:
I will get in contact with them myself, will keep you updated about the result.

Reply:
I will also chat to the hardware team.

Reply:
I have written an suggestion below